test-harness/src/test/groovy/com/netflix/conductor/test/integration/SubWorkflowSpec.groovy

```diff
@@ -461,13 +464,16 @@ class SubWorkflowSpec extends AbstractSpecification {
             tasks[2].status == Task.Status.COMPLETED
         }
 
+        and:
+        sweep(workflowInstanceId)
+
         then: "the workflow is in a COMPLETED state"
         with(workflowExecutionService.getExecutionStatus(workflowInstanceId, true)) {
             status == Workflow.WorkflowStatus.COMPLETED
             tasks.size() == 2
             tasks[0].taskType == 'integration_task_1'
             tasks[0].status == Task.Status.COMPLETED
-            tasks[1].taskType == 'SUB_WORKFLOW'
+            tasks[1].taskType == TASK_TYPE_SUB_WORKFLOW
```

Comment:
also verify the setting and resetting of the `subworkflowChanged` flag.

Reply:
Done